### DISCUSSION TITLE
No way to uninstall dbeaver if installed from tar.gz?

### DISCUSSION THREAD

**Discussion by otalado** [dbeaver/dbeaver#19700] [Q&A] [answered]
I installed dbeaver-ce to a **Raspberry PI 4 (debian bullseye)** from **dbeaver-ce-22.2.3-linux.gtk.aarch64-nojdk.tar.gz**
Now I need to uninstall/remove it. I found a lot of suggestions on how to do it: but they were either obsolete or not applicable. I (of course) browsed your docs, but while the docs are excellent, there is no mention of how to install or uninstall the dbeaver. I know what to do if it was installed by a snap or by apt package, but not if it was installed from tar.gz package.

I apologize if this is the wrong spot to ask this, but a possible alternative did not look any better to me.

**Comment by Destrolaric** [comment, marked as answer]
You can do it by simply deleting the application folder.
